src/server/api/trpc.ts

```diff
@@ -36,29 +36,29 @@ interface CreateContextOptions {
  *
  * @see https://create.t3.gg/en/usage/trpc#-serverapitrpcts
  */
-export const createInnerTRPCContext = async (opts: CreateContextOptions) => {
+export async function createInnerTRPCContext(opts: CreateContextOptions) {
```

Comment:
but why?

Reply:
Don't like them that's all. If something is a function it should be a function.